### DISCUSSION TITLE
CSS `media` queries in `ftd`

### DISCUSSION THREAD

**Discussion by gsalunke** [fastn-stack/fastn#795] [Ideas & RFCs]
Can we have CSS media queries in `ftd`. While working on [admint.io](https://admint.io/) project we realised if have media queries in `ftd` then it helps to create `px` to `px` matching UI as per client projects.

**Comment by amitu** [comment]
@gsalunke feature requests are to go as discussion in Ideas and RFCs. Issues is for things that are implemented but broken. Create a discussion and close this issue.

**Reply by amitu** [reply]
Nevermind, there was a button to do it, so I did it.
